package-lock.json

```diff
@@ -14,9 +14,9 @@
       "resolved": "https://registry.npmjs.org/align-text/-/align-text-0.1.4.tgz",
       "integrity": "sha1-DNkKVhCT810KmSVsIrcGlDP60Rc=",
       "requires": {
-        "kind-of": "3.2.2",
-        "longest": "1.0.1",
-        "repeat-string": "1.6.1"
+        "kind-of": "^3.0.2",
+        "longest": "^1.0.1",
+        "repeat-string": "^1.5.2"
```

Comment:
The fact that all these changes appeared in the package-lock.json threw me at first. Turns out it's a *cosmetic* difference between NPM 5 and NPM 6.

More discussion at https://github.com/npm/npm/issues/20891#issuecomment-395591772.